src/test/regress/greenplum_schedule

```diff
@@ -189,7 +189,7 @@ test: bb_memory_quota memconsumption
 
 test: tuple_serialization
 # Tests for replicated table
-test: rpt rpt_joins rpt_tpch rpt_returning
+test: rpt rpt_joins rpt_tpch rpt_returning bfv_dml_rpt
```

Comment:
why do we need a separate test file? Can't we add this test to some existing file (for ex. 'bfv_dml.sql') and have less files touched and less entities?

Reply:
I think, it is better to make separate tests, because big files is worse to understand than small.